gitgud/skills/parsing.py

```diff
@@ -1,6 +1,20 @@
 from copy import deepcopy
 
 
+def parse_solution(solution_path):
+    solution_commands = []
+
+    if not solution_path:
+        return solution_commands
+
+    solution_data = solution_path.read_text()
+    for command in solution_data.split('\n'):
+        if not command or command.strip()[0] == '#':
+            continue
+        solution_commands.append(command)
+    return solution_commands
```

Comment:
I think this whole method could probably be `self.solution_path.read_text().split()`. See other comments

Reply:
That'll work unless `solution.txt` doesn't exist for some reason. https://github.com/benthayer/git-gud/pull/187#discussion_r434074912

Reply:
Also, doing `split()` would split each command into each keyword, e.g. `git gud commit` would be processed as `["git", "gud", "commit"]`, and the shell would try to run each of those as individual commands. So, I'm leaving the function as `split('\n')`.

Reply:
`split('\n')` is totally fine